smart-frontend/app/src/store/adh/rules/rulesSlice.ts

```diff
@@ -75,10 +77,42 @@ const refreshRules = createAsyncThunk('adh/rules/refreshRules', async (_, thunkA
   thunkAPI.dispatch(loadRules());
 });
 
+const getActiveRulesCount = createAsyncThunk('adh/rules/getActiveRulesCount', async (_, thunkAPI) => {
+  try {
+    const collection = await AdhRulesApi.getRules({ ruleStates: [AdhRuleState.Active] });
+
+    return collection;
+  } catch (error) {
+    thunkAPI.dispatch(
+      showError({
+        message: getErrorMessage(error as RequestError),
+      }),
+    );
+    return thunkAPI.rejectWithValue(error);
+  }
+});
+
+const getAllRulesCount = createAsyncThunk('adh/rules/getAllRulesCount', async (_, thunkAPI) => {
+  try {
+    const collection = await AdhRulesApi.getRules({});
+
+    return collection;
+  } catch (error) {
+    thunkAPI.dispatch(
+      showError({
+        message: getErrorMessage(error as RequestError),
+      }),
+    );
+    return thunkAPI.rejectWithValue(error);
+  }
+});
+
 const createInitialState = (): AdhRulesSliceState => ({
   rules: [],
   totalCount: 0,
   loadState: LoadState.NotLoaded,
+  activeCount: 0,
+  allCount: 0,
```

Comment:
SliceState already contains `totalCount` field. what is`allCount` for?

Reply:
`totalCount` used for filtered items, `allCount` is for counter, shows all rules even they are filtered